Adds folder structure

doesitscript opened the pull request:
Addresses issue #1. 

Suggesting to use for presentation content format:

```
"YYYY-Presentations" 
```

Suggesting to use for subfolder content:

```
"MM - Topic (Presenter)"  
```
- Makes modification to suggested format for presentation content. 
- Allows for a single pane view of the years presentations.

_Note: If merging, please refer to issue that this closes. e.g. Closes #<number>_

_Not merging own work to allow for review._

doesitscript committed:
Creates folder structure for presentations

doesitscript committed:
Adds Speaker Bios

doesitscript committed:
Changed folder structure

Related issue #1

michaeltlombardi commented:
Couple questions and notes:

## Questions
1. Why are there gitignore files in every subfolder?
2. Is it preferable to have a subfolder in /Events/Meetups/2016 for each month and then the presentations of each month in that subfolder? As is, you end up with a third subfolder for March to hold the takeaways document or a takeaways document for every month in `/Events/Meetups/2016-Presentations`.
3. It looks like speaker bios ended up in a subfolder under presentations - is this intentional? If so, why?
4. This is nitpicky, but is it better to have the folder names capitalized or left all lowercase? It probably doesn't matter for people using windows machines but *nix boxes are case sensitive.

## Notes
- The `Automation-Manamement-Summit` folder needs to be renamed to `Automation-Management-Summit`
- I've updated #1 with an updated folder structure based on your inputs here.
- I have a strong preference for avoiding spaces in file and folder names wherever possible - it prevents mistyping and strange path problems on *nix systems and occasionally Windows ones as well. It also keeps URLs consistent.
- I have no real preference for naming month folders by name or by number. Whichever seems more clear should win.

Thanks for submitting the PR and doing the up-front work! I think if we can resolve the problems and address the concerns above this PR could be merged to master.

doesitscript committed:
Adds folders per #1 and #15

doesitscript commented:
Requested changes made with exception of changing away from CamelCase. 

### Justification

Too tired. <Sarcasm>Feel free to fork my fork, make a pull request to my fork, I will then review and merge changes... you get the picture </Sarcasm>

doesitscript commented on Events/Meetups/2016/April/agenda.md:
```diff
@@ -0,0 +1 @@
+#agenda info
```
ignore this.

michaeltlombardi committed:
Updated folder structure.

Reference [#1](https://github.com/stlpsug/stlpsug/issues/1)

Move all of folder structure to lower case with words separated by a dash.

Added skeleton files for all files identified in the Directory Structure document.

michaeltlombardi committed:
Updated Directory Structure doc

Added title and enclosed structure in backticks so markdown formats correctly in Github.

Renamed "contributions" to more sensical "contributing" - the file will give guidelines to help people contribute, not list contributions; github does that for us.

michaeltlombardi committed:
Updated readme

Added basic info for splash page, linking to main docs.

doesitscript committed:
Removes camelcase from folder structure

doesitscript committed:
Merge pull request #1 from michaeltlombardi/master

Updated Document Structure and Readme

doesitscript committed:
Add PSUG Icon

doesitscript commented:
Merged in your changes. Also I added the New PSUG icon.

michaeltlombardi commented:
Pinged the team on #14 since you're also addressing that in this PR.

michaeltlombardi commented:
Good job! We're accepting this PR now. :) Congrats on submitting the first one.
